README.md

```diff
@@ -7,61 +7,168 @@
 
-```swift
-var age = 19
+So far you've covered computer code that's of the form of directives and declarations; that is, "do this".  For example:
 
-if age < 21 {
-    print("You're not allowed to drink!")
+````Swift
+print("hello")   // hello
+// directive to print "hello"
+
+var x = 5 * 8
+// declaration of variable x
+
+let i = 12 >= 9
+// declaration of constant i
+````
+
+Most programs, however, need an additional component, and that component is Conditionals.  Conditionals are how you tell the program to take different behavior based upon conditions or results.  The simplest form of Conditional is just to do something if an expression is true.  We can express "if class is hard then study more" like this:
+
+````Swift
+if classIsHard {
+	studyMore()
 }
+````
 
-// prints "You're not allowed to drink!"
-```
+We can also specify an "else" clause.  For example, we can express "if class is hard then study more otherwise catch up on sleep" ike this:
 
-* Explain that an *if* statement can provide an alternative set of statements, known as an *else clause*, for situations when the *if* condition is *false*.
+````Swift
+if classIsHard() {
+	studyMore()
+} else {
+	catchUpOnSleep()
+}
+````
 
-```swift
-var myHeight = 50
+Are you begining to see the pattern?  Let's look at another one, maybe something more like an actual program, like "if the file exists then open it, otherwise tell the user it can't be found".  Let's see what it would look like in code:
 
-if myHeight > 60 {
-    print("You're free to go on the ride, enjoy!")
+````Swift
+let doesFileExist = findFileNamed("testfile")
+if doesFileExist == true {
+	openFileNamed("testfile")
 } else {
-    print("I'm sorry, you're too short to go on this ride today.")
+	reportError("testfile not found")
 }
+````
 
-// prints "I'm sorry, you're too short to go on this ride today."
-```
+Every Conditional effectively has both an ````if```` and an ````else```` part but depending upon how we style our code we can leave out the explicit ````else```` (which the compiler interprets as "if the condition is false then don't do anything").  For example, we could re-write the code above as follows:
 
-* Create/chain multiple *if* statements together to consider additional clauses:
+````Swift
+let doesFileExist = findFileNamed("testfile")
+if doesFileExist == false {
+	reportError("testfile not found")
+	return
+}
+openFileNamed("testfile")
+````
 
-```swift
-var temperatureInFahrenheit = 100
+Can you see how they'e variations of the same thing?  Let's look at another example.
 
-if temperatureInFahrenheit < 32 {
-    print("Snow! 🌨")
-} else if temperatureInFahrenheit >= 90 {
-    print("Very hot! ☀️")
+````Swift
+if rainingOutside == true {
+	takeUmbrella()
 } else {
-    print("Just right. 🌻")
+	// don't worry about taking the umbrella
```

Comment:
Maybe include another comment below that states no code is being excused here, just comments!

---

```swift
func takeUmbrella() {
    // grab umbrella
    print("I have my umbrella! ☂️")
}

var rainingOutside = false

if rainingOutside {
    takeUmbrella()
} else {
    // don't worry about taking the umbrella
    // no code is being excuted here, only comments!
}
```